programs/voter-stake-registry/Cargo.toml

```diff
@@ -21,31 +22,19 @@ test-sbf = []
 
 [dependencies]
 # a) for deployment use these
-anchor-lang = { version = "0.26.0", features = ["init-if-needed"] }
-anchor-spl = { version = "0.26.0" }
-# b) while testing, use below dependencies for debugging instead of above ones
-# anchor-lang = { git = "https://github.com/microwavedcola1/anchor.git", branch = "master-debug" }
-# anchor-spl = { git = "https://github.com/microwavedcola1/anchor.git", branch = "master-debug", features = ["governance"]}
-
-# The rev used for spl-governance must match what the fixture binary
-# programs/voter-stake-registry/tests/fixtures/spl_governance.so is built from.
+anchor-lang = { workspace = true, features = ["init-if-needed"] }
+anchor-spl = { workspace = true }
+solana-program = { workspace = true }
+static_assertions = { workspace = true }
 
 bytemuck = "1.9.1"
-
-# WARNING: Be careful about upgrading the spl-governance dependency!
```

Comment:
is this warning not relevant anymore?

Reply:
I don't think it was even the case for our instance